README.md

```diff
@@ -1,2 +1,50 @@
-# hello_world
-learning git
+# Learing Git
+
+## What is Git
+
+Git is a distributed version control system.  You can read all the details on line at
+
+https://git-scm.com/book/en/v2
+
+An interesting talk Linus gave about the motivations behind Git is here
+
+https://www.youtube.com/watch?v=4XpnKHJAok8
+
+Warning: Linus is kind of a git himself!
+
+## The least you need to know
+
+These are the commands you are going to use all the time:
+
+| Command | What it does? |
+|---------|---------------|
+| `git clone` | Copy repository (usually from GitHub) to your local machine |
```

Comment:
The `|------|` stuff here is defining a table in GFM - github flavored markdown.